assets/css/colors.scss

```diff
@@ -0,0 +1,10 @@
+$yellow: #FDC129;
+$light-turquoise: #EBF8F2;
+$apricot: #fbf2ea;
+$pink: #efb7b6;
+$brand-info: #4572C0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#4572C0" to be "#4572c0" (color-hex-case)](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5681062)